.github/workflows/_preview-create.yml

```
@@ -13,13 +13,16 @@ env:
   REACT_APP_AUTH0_AUDIENCE: https://dev-api.datamermaid.org
   REACT_APP_AUTH0_CLIENT_ID: 4AHcVFcwxHb7p1VFB9sFWG52WL7pdNm5
   REACT_APP_AUTH0_DOMAIN: datamermaid.auth0.com
+  REACT_APP_MAPBOX_ACCESS_TOKEN: pk.eyJ1IjoibWVybWFpZC1hZG1pbiIsImEiOiJjanh1djd1d20wMThoM2ZsaXZvdzZ3NGQxIn0.MrVUTQrUGJaTxjmn3h4udw
   REACT_APP_MERMAID_API: https://dev-api.datamermaid.org/v1
   REACT_APP_MERMAID_REFERENCE_LINK: https://dev-public.datamermaid.org/mermaid_attributes.xlsx
   REACT_APP_MERMAID_DASHBOARD_LINK: https://dev-dashboard.datamermaid.org
   REACT_APP_CORAL_ATLAS_APP_ID: v8nkxznp3m
   REACT_APP_MERMAID_WHATS_NEW_LINK: https://datamermaid.org/reef-stories/mermaid-v2-what-s-new
   REACT_APP_CAPTURE_GOOGLE_ANALYTICS: false
 
+ 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)